testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -133,18 +133,18 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I click the Add Product button
     And I wait until I see "SUSE Linux Enterprise Server with Expanded Support 7" product has been added
 
-@centos8_minion
+@rocky8_minion
   Scenario: Add SUSE Linux Enterprise Server with Expanded Support 8
     When I follow the left menu "Admin > Setup Wizard > Products"
     And I wait until I do not see "Loading" text
-    And I enter "RHEL or SLES ES or CentOS 8 Base" as the filtered product description
-    And I select "RHEL or SLES ES or CentOS 8 Base" as a product
-    Then I should see the "RHEL or SLES ES or CentOS 8 Base" selected
-    When I open the sub-list of the product "RHEL or SLES ES or CentOS 8 Base"
-    And I select "SUSE Linux Enterprise Server with Expanded Support 8" as a product
-    Then I should see the "SUSE Linux Enterprise Server with Expanded Support 8" selected
+    And I enter "Rocky Linux 8" as the filtered product description
+    And I select "Rocky Linux 8 x86_64" as a product
+    Then I should see the "Rocky Linux 8 x86_64" selected
+    When I open the sub-list of the product "Rocky Linux 8 x86_64"
+    And I select "SUSE Manager Client Tools for RHEL and ES 8 x86_64" as a product
+    Then I should see the "SUSE Manager Client Tools for RHEL and ES 8 x86_64" selected
     When I click the Add Product button
-    And I wait until I see "RHEL or SLES ES or CentOS 8 Base" product has been added
+    And I wait until I see "Rocky Linux 8 x86_64" product has been added
```

Comment:
No no.
We don't want that. Not for this PR at least.
We want to keep testing RES (RH Expanded Support) product, the one created and handled by SUSE, using the DVD image of a Rocky8.
We don't want to test Rocky8 Pure as the goal of this PR. Even if we might consider that other additional client in the future.

Reply:
**Clarifying**:
All RHEL-like products can be registered using their Pure SCC products or using RES (A special subscription created by SUSE to handle that kind of Distros).
We should cover both options, but for now, we want to assure to cover RES.

Reply:
Thanks for the clarificaiton. I reverted those changes.